changelog/may2025/2025-05-14-instances-removed-block-storage-volume-creation-removed.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Block Storage volume creation removed from the Instance API
+status: removed
+date: 2025-05-01
+category: compute
+product: instances
+---
+
+Since April 30th 2025, it has become impossible to **CREATE** new block volumes using the Instance API (or through associated developer tools). Customers can only list, mount, unmount or delete volumes through the Instance API. Block volume creation and management must go through the Block Storage API. [Learn more about the advantages of migrating from the Instance API to the [Block Storage API](https://www.scaleway.com/en/developers/api/block/) for managing block volumes and snapshots](/block-storage/reference-content/advantages-migrating-to-sbs/).
```

Comment:
```suggestion
Since April 30th 2025, it has become impossible to **CREATE** new block volumes using the Instance API (or through associated developer tools). Customers can only list, mount, unmount or delete volumes through the Instance API. Block volume creation and management must go through the Block Storage API. Learn more about the advantages of migrating from the Instance API to the [Block Storage API](https://www.scaleway.com/en/developers/api/block/) for [managing block volumes and snapshots](/block-storage/reference-content/advantages-migrating-to-sbs/).
```